wurst/objects/units/ShopObjects.wurst

```diff
@@ -34,3 +40,26 @@ public function HeroDefinition.setTotemShopInterface()
     ..setStockReplenishInterval(0)
     ..setIconGameInterface(Icons.bTNCancel)
     ..setUnitSoundSet("")
+
+@compiletime function createTotem()
+    let ABILITY_SELECT_UNITS_TOTEM = compiletime(UNIT_ID_GEN.next())
+
+    new AbilityDefinitionAlliedBuilding(ABILITY_SELECT_UNITS_TOTEM)
+    ..setActivationRadius(1, 1) // Small activation radius so it doesn't try selecting trolls
```

Comment:
Always indent `..` blocks.

Reply:
I based it off units.wurst which doesn't indent .. blocks but I can indent them